recipes/diligent-fx/all/conanfile.py

```diff
@@ -1,8 +1,12 @@
 import os
-from conans import ConanFile, tools, CMake
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile
+from conans import CMake
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import get, collect_libs, export_conandata_patches, apply_conandata_patches, rename
 import shutil
 
+required_conan_version = ">=1.53.0"
```

Comment:
```suggestion
required_conan_version = ">=1.52.0"
```
Not using 1.53.0 features.

Reply:
Done, please approve